test/acceptance/features/bindAppToService.feature

```diff
@@ -906,3 +906,49 @@ Feature: Bind an application to a service
                 ready: true
             """
         Then Secret "binding-request-backend-ann" is empty
+
+        Scenario: Bind an application to a service present in a different namespace
```

Comment:
Please use proper indentation ... like in https://github.com/pratikjagrut/service-binding-operator/pull/2 Thx

Reply:
@pratikjagrut after you fix the indentation above, squash the commits and pu a nice commit message... Thx

Reply:
@pmacik PTAL